cmd/lotus-chainwatch/main.go

```diff
@@ -26,6 +26,11 @@ func main() {
 				EnvVars: []string{"LOTUS_PATH"},
 				Value:   "~/.lotus", // TODO: Consider XDG_DATA_HOME
 			},
+			&cli.StringFlag{
+				Name:    "api",
+				EnvVars: []string{"FULLNODE_API_INFO"},
```

Comment:
Missing `LOTUS_API_INFO`.

Reply:
Per out of band discussion we are going to drop the requirement that there be 2 env vars for this, we will just look for `FULLNODE_API_INFO`